core/src/main/scala/com/microsoft/azure/synapse/ml/param/ArrayMapParam.scala

```diff
@@ -36,12 +40,12 @@ object ArrayMapJsonProtocol extends DefaultJsonProtocol {
       (kvp._1, convValue)
     })
   }
-
 }
 
 /** Param for Array of stage parameter maps. */
 class ArrayMapParam(parent: String, name: String, doc: String, isValid: Array[Map[String, Any]] => Boolean)
-  extends Param[Array[Map[String, Any]]](parent, name, doc, isValid) with WrappableParam[Array[Map[String, Any]]] {
+  extends Param[Array[Map[String, Any]]](parent, name, doc, isValid) with WrappableParam[Array[Map[String, Any]]]
+  with RWrappableParam[Array[Map[String, Any]]] {
```

Comment:
@mhamilton723 When you fix this in the other spot, let me know and I'll fix it here too.